apps/website-new/docs/zh/guide/start/npm-packages.mdx

```diff
@@ -0,0 +1,105 @@
+# npm 包
+
+本文档展示了 Module Federation 团队维护的所有 npm 包信息。
+
+
+## @module-federation/enhanced
+
+![](https://img.shields.io/npm/v/@module-federation/enhanced?style=flat-square&colorA=564341&colorB=EDED91)
```

Comment:
how about this ? 
```md
[![npm][npm]][npm-url]
[![node][node]][node-url]
![npm](https://img.shields.io/npm/dw/html-webpack-plugin.svg)

[npm]: https://img.shields.io/npm/v/html-webpack-plugin.svg
[npm-url]: https://npmjs.com/package/html-webpack-plugin

[node]: https://img.shields.io/node/v/html-webpack-plugin.svg
[node-url]: https://nodejs.org

```
![image](https://github.com/module-federation/core/assets/41466093/d6713c6d-c25b-4f6b-a12c-80ed15c7bd06)